gateway/ln-gateway/src/cln.rs

```diff
@@ -133,22 +132,51 @@ impl LnRpc for Mutex<cln_rpc::ClnRpc> {
     }
 }
 
+/// BOLT 4: https://github.com/lightning/bolts/blob/master/04-onion-routing.md#failure-messages
+/// 16399 error code reports unknown payment details.
+///
+/// TODO: We should probably use a more specific error code based on htlc processing fail reason
+fn htlc_processing_failure() -> serde_json::Value {
+    serde_json::json!({
+        "result": "fail",
+        "failure_message": "1639"
+    })
+}
```

Comment:
Can this be a `const`?

Reply:
Tried a const but the strings need to be statics at best. Didn't know how to make it a const